.github/workflows/ci.yml

```diff
@@ -31,8 +31,7 @@ jobs:
           key: ${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.erlang }}-${{ hashFiles('mix.lock') }}
       - uses: actions/cache@v4
         with:
-          # Don't cache PLTs based on mix.lock hash, as Dialyzer can incrementally update old ones
-          key: ${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.erlang }}-plt
+          key: ${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.erlang }}-${{ hashFiles('mix.lock') }}-plt
```

Comment:
It is still true that Dialyzer can incrementally update old PLTs, but this meant that for every CI run dialyzer needs to do this incremental update, because `actions/cache` doesn't save the updated plts if there was a cache hit.